doc/source/user_guide/computation.rst

```diff
@@ -648,6 +648,30 @@ from present information back to past information. This allows the rolling windo
 Currently, this feature is only implemented for time-based windows.
 For fixed windows, the closed parameter cannot be set and the rolling window will always have both endpoints closed.
 
+.. _stats.rolling_window.iter:
+
+Iteration over window:
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.1.0
+
+It is allowed to iterate over the Series and DataFrame with the ``rolling`` and ``expanding`` window are
```

Comment:
Maybe just:

` ``Rolling`` and  ``Expanding`` now support iteration.`

Reply:
shortened